devito/ir/iet/visitors.py

```diff
@@ -243,29 +244,54 @@ def visit_tuple(self, o):
 
     def visit_PointerCast(self, o):
```

Comment:
Note for reviewers: the content of this method will be subsequently changed by upcoming PRs (becoming simpler hopefully)